docs/development_guide/README.md

```diff
@@ -0,0 +1,55 @@
+# Development Guide Overview:
+
+This development guide provides information on the structure of the repository, testing infrastructure,
+style guides, and contributing guidelines. The hope is that this guide will help developers 
+create and maintain code that is consistent with the rest of the repository.
+
+## Repository Structure:
+
+The repository includes files such as the LICENSE and README.md files, which provide legal and informational 
+overviews of the repository. Other directories such as docker, docs, resources, scripts, 
+site, test, and wdl contain various files and directories that are important for 
+building and testing the software in the repository.  
+See [Repository Structure](./repo_structure.md) documentation for further 
+details and [Contributing Guidelines](#contributing-guidelines) for 
+information on how to contribute to the repository.
+
+## Workflow Scripts:
+
+All workflow scripts are located in the `/wdl` directory and are written in WDL 1.0 
+and intended for use with Google Cloud Platform via the scientific workflow engine, Cromwell.
+The WDL scripts are divided into three subdirectories: `tasks`, `structs`, and `pipelines`;
+then further divided by sequencing platform and analysis type.
+See [Repository Structure](./wdl_style_guide.md) for more information on directory structure.
+
+## Docker Containers:
+
+The WDL workflows in this repository are designed to be run using Docker containers. 
+This provides a number of advantages, including the ability to run the workflows on a 
+variety of platforms and the ability to isolate the workflows from each other.
+Many of these workflows use specialized containers that are built from the Dockerfiles in the 
+docker directory. The docker directory contains Dockerfiles and other scripts for 
+building containers for several tools. The docker containers are pushed to 
+Google Container Registry (GCR) called `us.gcr.io/broad-dsp-lrma`, for internal use. External audiences interested in running workflows
+using these containers should build and if needed push them to repository they have access to.
+
+## Testing:
+
+Scripts for running tests are located in the test directory. The test directory contains
+scripts for running tests using Tox. Tox is a Python-based test automation tool that 
+can be used to run tests in a variety of environments. The scripts are run through
+GitHub Actions, which are configured in the .github/workflows directory. The GitHub Actions
+are triggered by pushes to the repository and pull requests. 
+
+## Contributing Guidelines:
+
+Please adhere to the following best practices if contributing to this repository:
+
+1. Read Style Guide: Before making any changes to the code, it's important to read the style guide. The style guide contains information on how to write code that is consistent with the rest of the codebase. See [WDL Style Guide](./wdl_style_guide.md) and [Docker Style Guide](./docker_style_guide.md) for more information.
```

Comment:
Maybe put the first part of these sentences (before the ":") in bold, so people can see at a glance what the guidelines are.

Reply:
done